cmd/release-controller/sync.go

```diff
@@ -547,6 +547,26 @@ func (c *Controller) syncAccepted(release *Release) error {
 			}
 		}
 	}
+	// If bugzilla bugs were fixed by a PR approved by their QA contact, move bug to verified
+	if c.bugzillaVerifier != nil {
```

Comment:
This may have gotten lost in the previous PR, but this can't and should not belong here.  You need to move this to a side sync loop like audit (which is just a separate main controller loop att the top level) that gets notifications of changes to image streams but has a completely different work queue and lifecycle.  

With the current code structure, if any of this fails it will never get run again, and the go routines here actually make that worse because we could run syncAccepted fifty times in a second.

Your new loop at the top level is "ensure that every new accepted release has successfully marked all bugs fixed between those two levels as verified at least once, and then in the future it can be a no-op".  The two parts that are still missing are the "no-op" part - if you've completed successfully you need some indicator that is so (probably for now we'll add that as a verification field on the actual release tag) - and the control flow that doesn't ever run two coroutines at the same time.

While audit.go is more structured around running jobs, you can start with its flow by running the "verify bugs" function in its place, and if everything completes successfully annotating the release tag with an agreed on value (for now you can test with an arbitrary name and then when I see the code we can agree on what the actual value and name will be).  Then you skip any release that has that "already verified successfully", and retry any that don't have it.

Also, it looks like there's a logical gap here that you need to only be running this on image streams that are nightlies (CIs are considered insufficient to close bugs), so you'll need to add an attribute/field/struct on the Release configuration that allows only those inputs to be selected.  That's probably going to be adding a new struct to "ReleasePublish" which is an optional `*VerifyBugs` that will hold any configuration we need.  The presence of the struct will be required in your audit-like-loop to execute this code.

If you'd like to set up time on Monday to discuss this, poke me in slack and I'll clear some space on the calendar.